tests/unit/remoteworker/test_remote_worker.py

```diff
@@ -0,0 +1,126 @@
+import mock
+
+from unittest import TestCase
+
+from scripts.workers.remote_submission_worker import (
+    make_request,
+    get_message_from_sqs_queue,
+    delete_message_from_sqs_queue,
+    get_submission_by_pk,
+    get_challenge_phases_by_challenge_pk,
+    get_challenge_by_queue_name,
+    get_challenge_phase_by_pk,
+    update_submission_data,
+    update_submission_status,
+    return_url_per_environment,
+)
+
+
+@mock.patch("scripts.workers.remote_submission_worker.AUTH_TOKEN", "test_token")
+@mock.patch("scripts.workers.remote_submission_worker.requests")
+class MakeRequestTestClass(TestCase):
+
+    def test_make_request_get(self, mock_make_request):
+        url = "/test/url"
+        headers = {"Authorization": "Token test_token"}
+        make_request(url, "GET")
+        mock_make_request.get.assert_called_with(url=url, headers=headers)
+
+    def test_make_request_put(self, mock_make_request):
+        url = "/test/url"
+        headers = {"Authorization": "Token test_token"}
+        data = {"test": "data"}
```

Comment:
@KhalidRmb you can move the redundant variables in a common base class

Reply:
@Ram81 Sure. Will make the changes.